src/txn/txn.c

```diff
@@ -878,6 +878,77 @@ __txn_append_hs_record(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor, WT_ITEM *
     return (ret);
 }
 
+/*
+ * __txn_commit_timestamps_usage_check --
+ *     Print warning messages when encountering surprise timestamp usage.
```

Comment:
The word `surprise` is not right. Some possibilities: `unexpected` or `surprising` or `incorrect`.

Reply:
1a9d666